src/ngx_http_lua_rewriteby.c

```diff
@@ -164,24 +171,74 @@ ngx_http_lua_rewrite_handler(ngx_http_request_t *r)
 
 
 ngx_int_t
-ngx_http_lua_rewrite_handler_inline(ngx_http_request_t *r)
+ngx_http_lua_rewrite_handler_sets(ngx_http_request_t *r)
```

Comment:
I don't like the `_sets` suffix, which is too vague. Let's come up with a better name.